README.md

```diff
@@ -7,9 +7,61 @@
 An [Apache 2.0](https://github.com/allenai/allennlp/blob/master/LICENSE) NLP research library, built on PyTorch,
 for developing state-of-the-art deep learning models on a wide variety of linguistic tasks.
 
-## Quickstart
+## Running AllenNLP
+
+The preferred way to install AllenNLP is via `pip`.
+
+### Setting up a virtual environment
+
+[Conda](https://conda.io/) can be used set up a virtual environment
+with the version of Python required for AllenNLP and in which you can
+sandbox its dependencies:
+
+1.  [Download and install Conda](https://conda.io/docs/download.html).
+
+2.  Create a Conda environment with Python 3.6
+
+    ```
+    conda create -n allennlp python=3.6
+    ```
+
+3.  Activate the Conda environment.  (You will need to activate the Conda environment in each terminal in which you want to use AllenNLP.
+
+    ```
+    source activate allennlp
+    ```
+
+4. Install AllenNLP in your environment.
```

Comment:
I think you can remove this step - it's not part of "setting up a virtual environment", it's part of the next section.